Documentation/MigrateToComposer/BestPractices.rst

```diff
@@ -28,7 +28,10 @@ Define a local path repository
 ------------------------------
 
 Create a directory :file:`packages` in your project root folder and define
-this folder as a repository of type "path" in your :file:`composer.json`::
+this folder as a repository of type "path" in your :file:`composer.json`:
+
+.. code-block:: json
+   :caption: typo3_root/composer.json
```

Comment:
```suggestion
   :caption: composer.json
```

Reply:
I want to clarify here the this is the composer.json at the root of your TYPO3 project, not one in an extension or some such. I believe this is more clear in my version then in yours where it could be any composer.json.

Reply:
Then `/composer.json` should be good enough.